src/components/People/PeopleData.js

```diff
@@ -0,0 +1,55 @@
+import { StaticQuery } from 'gatsby'
+import React from 'react'
+
+export const filterVisiblePersons = (data, type, hashtag = '') => {
+  return data.filter(({ label, tracklist }) => {
+    if (label === type) {
+      if (type === 'mentor') {
+        return hashtag.includes(tracklist.toLowerCase())
+      }
+      return true
+    }
+    return false
+  })
+}
```

Comment:
Maybe this can me made into an expression something like this?

```
export const filterVisiblePersons = (data, type, hashtag = '') => data.filter(({ label, tracklist }) => 
      label === type && type === 'mentor' && hashtag.includes(tracklist.toLowerCase())
```

Just an idea, otherwise LGTM 🚀

Reply:
only mentor has the hashtag requirement

Reply:
but it can be still done like this expression

Reply:
fixed